ui/settingsDeviceClassConfig.glade

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <!-- Generated with glade 3.22.1 -->
 <interface>
-  <requires lib="gtk+" version="3.0"/>
+  <requires lib="gtk+" version="3.18"/>
```

Comment:
Downgrade gtk+ library for backwards compatibility.

Reply:
I'm not sure this matters given that we already require GTK >= 3.18, but I've changed it back for now.

Reply:
Ahh then it would not have mattered. We had a problem a while back with a too new version string here causing issues. We should maybe make these consistent in a future update.